platforms/jetson/jetson_adaptor.go

```diff
@@ -0,0 +1,291 @@
+package jetson
+
+import (
+	"errors"
+	"fmt"
+
+	"sync"
+
+	multierror "github.com/hashicorp/go-multierror"
+	"gobot.io/x/gobot"
+	"gobot.io/x/gobot/drivers/i2c"
+	"gobot.io/x/gobot/drivers/spi"
+	"gobot.io/x/gobot/sysfs"
+)
+
+// Adaptor is the Gobot Adaptor for the Jetson Nano
+type Adaptor struct {
+	mutex              *sync.Mutex
+	name               string
+	revision           string
+	digitalPins        map[int]*sysfs.DigitalPin
+	pwmPins            map[int]*PWMPin
+	i2cDefaultBus      int
+	i2cBuses           [2]i2c.I2cDevice
+	spiDefaultBus      int
+	spiDefaultChip     int
+	spiDevices         [2]spi.Connection
+	spiDefaultMode     int
+	spiDefaultMaxSpeed int64
+	JSBlasterPeriod    uint32
```

Comment:
What is the meaning of "JSBlasterPeriod" name? IMO it is something like "pwmDefaultPeriod". Do we need to expose it? If the intention is to allow change of period, it would be better to implement a method "SetPeriod()", which sets the period directly and make the default value a constant.

Reply:
"JSBlasterPeriod" is simply mean "pwmPeriod". suggested name  "pwmDefaultPeriod" is correct.
I think this variable is expose according to Jetson Nano PWM tech spec, period enable range 200hz - 300khz. 
 fixed it [b04ecaa](https://github.com/hybridgroup/gobot/pull/799/commits/b04ecaa329975a51b30940a3188065a1a409ddc8), but i no confidence.
if this fix is wrong approach, teach me how do i fix it.